features/models.py

```diff
@@ -297,6 +304,11 @@ class Meta:
         verbose_name_plural = _("opening hours periods")
         ordering = ("id",)
 
+    def __str__(self):
+        return ", ".join(
+            [str(oh) for oh in self.opening_hours.all().order_by("day", "opens")]
```

Comment:
I don't think this should be here?

Reply:
I don't think so either. I wonder how it got into the branch...